### PR TITLE
open governance statement

### PR DESCRIPTION
**Pull request by wmorgan**

### CODE REVIEW

**Comment by wmorgan** [comment]
CI failure is a false positive

File: linkerd.io/content/blog/linkerd-and-open-governance.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+And we want YOU to join us._
+
+If you've been following Linkerd for a while, this should come as no surprise.
+This is all stuff we've said before. But in this post, I wanted to add a little
+more personal context.
+
+I wear two hats when it comes to Linkerd. I am one of the maintainers of the
+project. I am also the CEO of [Buoyant](https://buoyant.io). Buoyant created
+Linkerd in 2016, submitted it to the CNCF way back in the dark ages of 2017
+(when the CNCF only had 4 projects!). Buoyant continues to be the primary the
```
Comment (alpeb) [inline linkerd.io/content/blog/linkerd-and-open-governance.md]: `the primary the`

File: linkerd.io/content/blog/linkerd-and-open-governance.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+primary sponsor of the project, and to date, the majority of code in Linkerd
+come from folks who have been paid by Buoyant for their time and energy. In
```
Comment (alpeb) [inline linkerd.io/content/blog/linkerd-and-open-governance.md]: `comes`

File: linkerd.io/content/blog/linkerd-and-open-governance.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+fact, I take great pride in the fact that Buoyant has been able to find great
+people in the Linkerd community, like Alejandro, Ivan, Zahari, Sean, Carl, and
```
Comment (alpeb) [inline linkerd.io/content/blog/linkerd-and-open-governance.md]: :heart:

Comment (wmorgan) [reply linkerd.io/content/blog/linkerd-and-open-governance.md]: Right back at ya! 😻

File: linkerd.io/content/blog/linkerd-and-open-governance.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+maintainers is fundamental to the long-term health of an open source project.
+And we want YOU to join us._
+
+If you've been following Linkerd for a while, this should come as no surprise.
+This is all stuff we've said before. But in this post, I wanted to add a little
+more personal context.
+
+I wear two hats when it comes to Linkerd. I am one of the maintainers of the
+project. I am also the CEO of [Buoyant](https://buoyant.io). Buoyant created
+Linkerd in 2016, submitted it to the CNCF way back in the dark ages of 2017
```
Comment (klingerf) [inline linkerd.io/content/blog/linkerd-and-open-governance.md]: Maybe "...2016, _and_ submitted it..."

File: linkerd.io/content/blog/linkerd-and-open-governance.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+(when the CNCF only had 4 projects!). Buoyant continues to be the primary
+primary sponsor of the project, and to date, the majority of code in Linkerd
```
Comment (klingerf) [inline linkerd.io/content/blog/linkerd-and-open-governance.md]: Two `primary` here -- one at the end of line 32, and one at the beginning of line 33.

File: linkerd.io/content/blog/linkerd-and-open-governance.md (hunk 3 above)
Comment (klingerf) [reply linkerd.io/content/blog/linkerd-and-open-governance.md]: Consider linking these to names to Github. Something like:

```
people in the Linkerd community, like [Alejandro](https://github.com/alpeb),
[Ivan](https://github.com/ihcsim), [Zahari](https://github.com/zaharidichev),
[Sean](https://github.com/seanmonstar), [Carl](https://github.com/carllerche),
and ...
```

**Pull request merged**